.github/workflows/deploy.yml

```diff
@@ -78,7 +78,7 @@ jobs:
       run: pipx run nox -s prepare
 
     - name: Build distributions
-      run: pipx run build --sdist --wheel
+      run: pipx run build
```

Comment:
The default for `build` is to build sdist and wheel, so can drop these extra arguments.

Reply:
Sure, but being explicit is like documentation for someone reading the code. (Though it depends on what points you want to emphasize for the reader.)